src/css/components/files/_c-test-results.scss

```diff
@@ -0,0 +1,9 @@
+.c-test-results {
+  @include border_style(0.1em, 0px, rgba(204, 204, 204, 1.0));
+  margin-bottom: 10px;
+  padding: 1px;
+  overflow-y: scroll;
+  -webkit-box-shadow: -2px 2px 6px 2px rgba(0,0,0,0.14);
```

Comment:
You should ideally prefix in your build process. Doing it manually leaves chances of missing out on prefixing certain rules. Checkout `gulp-autoprefixer`.